docs/Hooks.md

```diff
@@ -232,6 +232,26 @@ fastify.addHook('onRequest', (request, reply, done) => {
 })
 ```
 
+If you are sending a response without `await` on it, make sure to always
```

Comment:
(GitHub won't let me add a comment to line 213)

On line 213 I think we should have more detail about _why_ these rules are in place and what is actually happening when they are followed. Even with the addition here on line 235 it's quite ambiguous as to what is going on.

Reply:
I've pushed some new text. Please check it out.